README.md

```diff
@@ -67,9 +67,8 @@ pudl_archiver --datasets {list_of_datasources}
 
 This command will download the latest available data and create archives for each
 requested datasource requested. The supported datasources include `censusdp1tract`,
-`eia_bulk_elec`, `eia176`, `eia191`, `eia757a`,`eia860`, `eia860m`, `eia861`, `eia923`,
-`eia930`, `eiaaeo`, `eiawater`, `epacems`, `epacamd_eia`, `ferc1`, `ferc2`, `ferc6`,
-`ferc60`, `ferc714`, `nrelatb`, `phmsagas`, `mshamines`.
+`eia_bulk_elec`, `eia176`, `eia191`, and many more; see the full list with
+`pudl_archiver --list`.
```

Comment:
I would keep our core datasets in there like `eia860`, `eia923`, `ferc1`, and `epacems` instead of `eia_bulk_elec` etc.

Reply:
and maybe say `see the full list of available datasets`